examples/ethernet-rtic-stm32h747i-disco.rs

```diff
@@ -153,15 +152,15 @@ const APP: () = {
         let mut link_led = gpioi.pi14.into_push_pull_output(); // LED3
         link_led.set_high().ok();
 
-        let _rmii_ref_clk = gpioa.pa1.into_alternate_af11().set_speed(VeryHigh);
-        let _rmii_mdio = gpioa.pa2.into_alternate_af11().set_speed(VeryHigh);
-        let _rmii_mdc = gpioc.pc1.into_alternate_af11().set_speed(VeryHigh);
-        let _rmii_crs_dv = gpioa.pa7.into_alternate_af11().set_speed(VeryHigh);
-        let _rmii_rxd0 = gpioc.pc4.into_alternate_af11().set_speed(VeryHigh);
-        let _rmii_rxd1 = gpioc.pc5.into_alternate_af11().set_speed(VeryHigh);
-        let _rmii_tx_en = gpiog.pg11.into_alternate_af11().set_speed(VeryHigh);
-        let _rmii_txd0 = gpiog.pg12.into_alternate_af11().set_speed(VeryHigh);
-        let _rmii_txd1 = gpiog.pg13.into_alternate_af11().set_speed(VeryHigh);
+        let rmii_ref_clk = gpioa.pa1.into_alternate_af11();
+        let rmii_mdio = gpioa.pa2.into_alternate_af11();
+        let rmii_mdc = gpioc.pc1.into_alternate_af11();
+        let rmii_crs_dv = gpioa.pa7.into_alternate_af11();
+        let rmii_rxd0 = gpioc.pc4.into_alternate_af11();
+        let rmii_rxd1 = gpioc.pc5.into_alternate_af11();
+        let rmii_tx_en = gpiog.pg11.into_alternate_af11();
+        let rmii_txd0 = gpiog.pg13.into_alternate_af11();
```

Comment:
`pg12` and `pg13` were incorrect here I think

Reply:
aha, they were swapped indeed